packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -38,4 +70,177 @@ Content-Type: application/octet-stream;
   @post
   @route("/mixed-parts")
   op basic(@header contentType: "multipart/form-data", @body body: MultiPartRequest): NoContentResponse;
+
+  @scenario
+  @scenarioDoc("""
+Expect request:
+```
+POST /upload HTTP/1.1
+Content-Length: 428
+Content-Type: multipart/form-data; boundary=abcde12345
+
+--abcde12345
+Content-Disposition: form-data; name="id"
+Content-Type: text/plain
+
+123
+--abcde12345
+Content-Disposition: form-data; name="address"
+Content-Type: application/json
+
+{
+  "city": "X"
+}
+--abcde12345
+Content-Disposition: form-data; name="profileImage"; filename="<any-name-is-ok>"
+Content-Type: application/octet-stream
+
+{…file content…}
+--abcde12345--
+Content-Disposition: form-data; name="previousAddresses"
+Content-Type: application/json
+
+[{
+  "city": "Y"
+},{
+  "city": "Z"
+}]
```

Comment:
My understand is for json object it should be a JSON string.

Reply:
Yeah, when you print a JSON string, it is like the up format.